### ISSUE TITLE
Wrong publish order?

### ISSUE THREAD

**Issue by ob1** [wix/stylable#1704]
Had a build fail on this:
```
error /home/builduser/work/c1dd1064fea009a6/node_modules/wix-ui-core: Command failed.
Exit code: 1
Command: npm install @stylable/dom-test-kit --no-save
Arguments: 
Directory: /home/builduser/work/c1dd1064fea009a6/node_modules/wix-ui-core
Output:
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @stylable/runtime@^4.0.2.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget 
npm ERR! notarget It was specified as a dependency of '@stylable/dom-test-kit'
npm ERR! notarget 
```

I saw that these packages were just published around the time of the build.

Looking at https://github.com/wix/stylable/runs/2113956012, I saw that `@stylable/dom-test-kit` was published before its dependency `@stylable/runtime`, and we just happened to catch them in the middle.

Perhaps the publish order should reflect the dependency chain

**Comment by AviVahl** [comment]
Looks like an issue with pleb's package sorting. Will check.

**Comment by AviVahl** [comment]
Got a tiny reproduction going thanks to @tomrav :+1: 
Will get it sorted for future publishes.

**Comment by AviVahl** [comment]
Issue has been fixed in [pleb@3.4.2](https://github.com/wixplosives/pleb/releases/tag/v3.4.2). :)
